packages/shared/src/helpers/mapToGuidelineDesignTokens.ts

```diff
@@ -119,10 +121,14 @@ const transformStringValues = (key: string, cssStyles: TokenValues, value: strin
 export const mapToGuidelineDesignTokens = (dataToTransform: Partial<Record<string, DesignTokenProperties>>) => {
     const transformedDesignTokens: TransformedDesignTokens = {};
     const enrichedDataToTransform = provideDefaultCalloutColors(dataToTransform);
+    const mainFontFamily =
+        (dataToTransform['main_font'] as Record<DesignTokenPropertiesEnum, string>)?.family === 'default'
```

Comment:
couldn't the above logic be reused here?

Reply:
refactored & tests added